arbitrator/jit/src/prepare.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022-2024, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
```

Comment:
Should we be sharing the code in this file with the similar code in the non-jit prover?